xbmc/platform/darwin/ios-common/AnnounceReceiver.mm

```diff
@@ -169,6 +165,30 @@ void AnnounceBridge(ANNOUNCEMENT::AnnouncementFlag flag,
       }
     }
 
+    // Live TV properties
+    if (g_application.CurrentFileItem().IsPVRChannel())
+    {
+      auto epg_now = g_application.CurrentFileItem().GetPVRChannelInfoTag()->GetEPGNow();
+      item[@"epg_title"] = @(epg_now->Title().c_str());
```

Comment:
I think a more generic approach would be better.

Fill the artist/title dict entries with the data you want here instead of using a "new" dict entry epg_title.

So if the epg entry has a title, set that to item[@"title"], and item[@"artist"] to the channel name, otherwise, just set item[@"title"] to the channel name.

im not even sure where the channel name comes from in the current code. couldnt find it at a quick glance, but its obviously being populated somewhere